src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.html

```diff
@@ -87,5 +87,11 @@ <h6>
         <h5><span jhiTranslate="artemisApp.fileUploadExercise.assessedSubmission">Your assessed submission</span>:</h5>
         <jhi-file-upload-result [result]="result"></jhi-file-upload-result>
     </div>
-    <jhi-complaint-interactions *ngIf="fileUploadExercise" [result]="result" [participation]="participation" [exercise]="fileUploadExercise"> </jhi-complaint-interactions>
+    <jhi-complaint-interactions
+        *ngIf="fileUploadExercise && fileUploadExercise.course?.complaintsEnabled"
+        [result]="result"
+        [participation]="participation"
+        [exercise]="fileUploadExercise"
```

Comment:
Is `result` the same as `participation.result` and `exercise` the same as `participation.exercise`?

Reply:
I suggest to investigate and potentially refactor this in a different PR.